articles/iot-hub/iot-hub-message-enrichments-overview.md

```diff
@@ -83,6 +83,8 @@ To try out message enrichments, see the [message enrichments tutorial](tutorial-
 
 * Message enrichments don't apply to digital twin change events.
 
+* Modules do not inherit twin tags from their corresponding devices. Enrichments for messages originating from device modules (for example from IoT Ege modules) must use the twin tags that are set on the module twin.
```

Comment:
You have IoT Ege here instead of IoT Edge.

Reply:
Ahh good catch